frameworks/Nim/scorper/scorper_bench.nim

```diff
@@ -0,0 +1,122 @@
+import scorper,macros,random,json,strformat,os,cgi,strutils,algorithm
+import amysql
+import ./scorper_bench/fortune_view
+
+const DB_DRIVER = getEnv("DB_DRIVER", "mysql")
+const DB_HOST = getEnv("DB_HOST", "tfb-database:5432")
```

Comment:
I don't know what the issue is atm, but this DB_HOST var still doesn't make sense to me. The toolset is spinning up a mysql database on port 3306 because that's what you put in your benchmark_config. This is the postgres port. I can't be sure by the issue isn't the server stopping because it can't secure a connection to the database, so can you remove all the code that isn't being used.

Reply:
sure, am working on this, It was my mysql driver not successful connected due to it only accept ip address , I print the url when py call verify , it first runs into /db test , I never thought that it pick /db entry first.